packages/nimble-components/src/table-column/icon/tests/table-column-icon-matrix.stories.ts

```diff
@@ -80,6 +82,19 @@ const component = (): ViewTemplate => html`
             <${mappingIconTag} key="1" text="One" icon="${iconCheckTag}" severity="warning"></${mappingIconTag}>
             <${mappingIconTag} key="2" text="Two" icon="${iconCheckTag}" severity="error"></${mappingIconTag}>
         </${tableColumnIconTag}>
+        <${tableColumnIconTag}
+            field-name="code"
+            key-type="number"
+            width-mode="${_ => TableColumnMappingWidthMode.iconSize}"
```

Comment:
Shouldn't need a binding for just a constant

```suggestion
            width-mode="${TableColumnMappingWidthMode.iconSize}"
```